### ISSUE TITLE
Update to jQuery 3.3.1

### ISSUE THREAD

**Issue by coliff** [matcornic/hugo-theme-learn#206]
It'd be good to update to jQuery 3.3.1.  jQuery 2.x is no longer supported and has some XSS issues (https://snyk.io/test/npm/jquery/2.2.4)

I've tested this theme with jQuery 3.3.1 and it works correctly.

**Comment by matalo33** [comment]
Would you be able to submit a PR for this?

If you do, please give a URL to where you obtained jquery from. Preferably a link to a specific version, not just 'latest'. This is so I can validate the file hasn't been modified before merging 👍

**Comment by coliff** [comment]
Sure thing. I'll do that shortly. I was thinking of suggesting that we load jQuery from a CDN with a local fallback - like HTML5Boilerplate recommends.

It'd be like this:
```
  <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
  <script>window.jQuery || document.write('<script src="js/jquery-3.3.1.min.js"><\/script>')</script>
```

REF: https://github.com/h5bp/html5-boilerplate/blob/6.1.0/dist/index.html

What do you think?

**Comment by matalo33** [comment]
Closed by #199

**Comment by matalo33** [comment]
I like the idea of using CDN but I remember having a conversation about this with @matcornic in the past... he wanted the site to remain functional even in an "internal" or "offline" network. 

This option of trying CDN and then falling back could work. We should check whether there's a noticeable delay while an 'offline' client fails or times out reaching the CDN.

Also it would make sense to apply this change to all libraries loaded by the theme which are available via a CDN.

**Comment by coliff** [comment]
Great - thanks for confirming @matcornic - I'll open a PR shortly.

**Comment by blackmichael** [comment]
@coliff Is there any progress on this?

**Comment by coliff** [comment]
This is dealt with PR #237 which I hope will be merged.

**Comment by matalo33** [comment]
Closed by #237
